tests/DTO/DefaultProxyFeatureTest.php

```diff
@@ -10,7 +10,7 @@ final class DefaultProxyFeatureTest extends TestCase
 {
     public function testGetProperties()
     {
-        $instance = new DefaultProxyFeature(['name' => 'test', 'enabled' => true, 'impression_data' => true, 'variant' => ['name' => 'someVariant', 'enabled' => true, 'payload' => ['type' => 'string', 'value' => 'test']]]);
+        $instance = new DefaultProxyFeature(['name' => 'test', 'enabled' => true, 'impressionData' => true, 'variant' => ['name' => 'someVariant', 'enabled' => true, 'payload' => ['type' => 'string', 'value' => 'test']]]);
```

Comment:
There should also be some tests testing that both the old and new names work.

Reply:
:+1: this should do it: 07b6c4f43985ea7fd96c758cbde29322c334a8af